dojo/tools/bugcrowd_api/api_client.py

```diff
@@ -0,0 +1,104 @@
+import requests
+from urllib.parse import quote
+
+from dojo.models import Tool_Type
+
+
+class BugcrowdAPI:
+    """
+    A simple client for the bugcrowd.io API
+    """
+
+    bugcrowd_api_url = "https://api.bugcrowd.com"
+
+    def __init__(self, tool_config):
+        tool_type, _ = Tool_Type.objects.get_or_create(name='Bugcrowd API')
```

Comment:
Shouldn't we use `get` and generate a notification if there is no configuration? Better than creating an empty one that will fail a few lines below

Reply:
Ah ! I copied over from the edgescan importer I think, so I didn't really know these other functions

Reply:
forget my comment , this part is good.